samples/net/openthread/coap_client/README.rst

```diff
@@ -24,34 +25,48 @@ User interface
 
 The following board buttons are used by this sample:
 
-BUTTON 1:
-   Pressing results in sending a unicast LIGHT_TOGGLE message to the /light resource on a paired device. If no device is paired with the specific client node, pressing the button has no effect.
+Button 1:
+  Pressing results in sending a unicast LIGHT_TOGGLE message to the /light resource on a paired device. If no device is paired with the specific client node, pressing the button has no effect.
 
-BUTTON 2:
-    Pressing results in sending a multicast LIGHT_ON or a LIGHT_OFF message (alternatively) to the /light resource. Sending a LIGHT_ON/LIGHT_OFF message instead of LIGHT_TOGGLE allows to synchronize the state of the LEDs on several server nodes.
+Button 2:
+  Pressing results in sending a multicast LIGHT_ON or a LIGHT_OFF message (alternatively) to the /light resource. Sending a LIGHT_ON/LIGHT_OFF message instead of LIGHT_TOGGLE allows to synchronize the state of the LEDs on several server nodes.
 
-BUTTON 3:
-    Pressing results in sending a multicast pairing request to the /provisioning resource.
+Button 3:
+  Pressing results in sending a multicast pairing request to the /provisioning resource.
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/net/openthread/coap_client`
+
+.. include:: /includes/build_and_run.txt
 
 
 Testing
 =======
 
-After programming the sample to your board, test it by performing the following steps:
+After building the sample, test it by performing the following steps:
+
+#. Program one or more boards with the Thread Simple CoAP Client and the Thread Simple CoAP Server.
+#. Turn on the Simple CoAP Server or Simple CoAP Client node and wait for a while.
+#. Turn on any of the other nodes. They will enter the network as Children, and Simple CoAP Client or Simple CoAP Server nodes will gradually become Routers.
+#. You can control LED 4 on every server node by pressing Button 2 on any client node.
+#. Pair a client with a server. Press Button 4 on a server node to enable pairing. Then, press Button 3 on any client node to pair the two nodes. You can now control the LED 4 on this server node by pressing Button 1 on the client node.
+
+.. tip::
+    You can connect to any of the Simple CoAP Server or Simple CoAP Client nodes through a serial port and run OpenThread CLI commands. The application uses the following UART settings:
```

Comment:
Isn't this covered by How to connect with PuTTY? Can we link there instead of providing this information here? https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_testing.html